Inc/orbtraceIf.h

```diff
@@ -16,14 +16,14 @@
     #include <sys/ioctl.h>
     #include <libusb.h>
     #include <termios.h>
+#elif defined LINUX
+    #include <libusb-1.0/libusb.h>
```

Comment:
```suggestion
    #include <libusb.h>
```
Consider dropping the subfolder -- it is not needed, if Makefile + pkg-config, or meson, is used as the build system. This allows simplifying most of nested macros dispatch in here and other places. I tested building on Linux. Git annotate points to early Makefile from 2017 without pkg-config, where it could be attributed to pointing under `/usr/include` ad-hoc.